src/expressjs/routes/products/index.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import * as products from '../../controllers/products';
+
+const productRouter = express.Router();
+
+productRouter.route('/').get(products.getAllProducts);
```

Comment:
as you're using `.route()` syntax, you can chain your middlewares for different request types like
```
productsRouter.route('/')
  .get(handlerForGet)
  .post(handlerForPost)
```
That's the reason for using .route() - chain all the handlers

Reply:
Thanks, updated code.